common/python/crypto_utils/crypto_utility.py

```diff
@@ -12,51 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-utility.py -- general helper function
-"""
 import base64
 import os
-import utility.file_utils as putils
 import utility.hex_utils as hex_utils
-import crypto.crypto as crypto
-import config.config as pconfig
+import crypto_utils.crypto.crypto as crypto
 import logging
 
 logger = logging.getLogger(__name__)
 
-TCFHOME = os.environ.get("TCF_HOME", "../../")
-# No of bytes of encrypted session key to encrypt data
-NO_OF_BYTES = 16
-
-
-def create_error_response(code, jrpc_id, message):
-    """
-    Function to create error response
-    Parameters:
-        - code: error code enum which corresponds to error response
-        - jrpc_id: JRPC id of the error response
-        - message: error message which corresponds to error response
-    """
-    error_response = {}
-    error_response["jsonrpc"] = "2.0"
-    error_response["id"] = jrpc_id
-    error_response["error"] = {}
-    error_response["error"]["code"] = code
-    error_response["error"]["message"] = message
-    return error_response
-
-
-# -----------------------------------------------------------------------------
-def strip_begin_end_key(key):
```

Comment:
This function can be kept in crtypo_utils as its doing operation specifically for public key.  Thsi function could be renamed as strip_begin_end_public_key

Reply:
Ok